packages/table/src/regions.ts

```diff
@@ -528,6 +528,20 @@ export class Regions {
         return regionGroups;
     }
 
+    public static regionArraysEqual(regionsA: IRegion[], regionsB: IRegion[]) {
+        if (regionsA === undefined && regionsB === undefined) {
+            return true;
+        } else if (regionsA === null && regionsB === null) {
+            return true;
+        } else if (regionsA == null || regionsB == null || regionsA.length !== regionsB.length) {
+            return false;
+        } else {
+            return regionsA
+                .map((regionA, i) => Regions.regionsEqual(regionA, regionsB[i]))
+                .every((isEqual) => isEqual);
```

Comment:
please just replace `map` with `every`. all `Array` iterators receive the same three args. no need to create a new array of booleans.